### ISSUE TITLE
Need black/white list feature like original maximus

### ISSUE THREAD

**Issue by benbarbour** [wilfm/GnomeExtensionMaximusTwo#4]
I really only want chrome and gnome-terminal to undecorate.

This is also a useful setting as an easy workaround for any future app that this doesn't play well with for whatever reason - just blacklist it.

**Comment by negesti** [comment]
I have added a preferences widget that makes it possible to disable Maximus for application based on the window title (contains)

**Comment by desyncr** [comment]
@negesti Thanks for these changes (btw [here](https://github.com/negesti/GnomeExtensionMaximusTwo/commit/735e7355edf2310d4ca18232a989a2354f21f2bd) are the changes if anybody want to test them). I'll try it out later on.

**Comment by desyncr** [comment]
@negesti Hey would you mind making a PR? The changes work but some things could be enhanced (like not having to restart the shell to have the changes applied etc)

**Comment by negesti** [comment]
added a settings listener, restart is no longer required

**Comment by wilfm** [comment]
A preference widget will be a great thing to implement, I think I'll produce a development branch soon which will feature it. 

BTW for the orginal issue of `I really only want chrome and gnome-shell to undecorate.`, you can disable 'Use system title bar and borders' in Chrome settings (`chrome://settings#appearance-section`)

**Comment by negesti** [comment]
Already created a pull request for that.

**Comment by wilfm** [comment]
Yes I know :) I've just got to update it for the new changes. What version of Gnome shell are you using?

**Comment by negesti** [comment]
No offense
I'm using Fedora 22 with gnome-shell 3.16  (3.16.3)
